build_stats.sh

```diff
@@ -0,0 +1,40 @@
+#! /bin/sh
+
+set -e
+
+HERE_DIR="$(cd "$(dirname "$1")"; pwd)/$(basename "$1")"
```

Comment:
FWIW I've been testing with `sh` rather than `bash` and this seems to be working